community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/StartsWithAcceptanceTest.scala

```diff
@@ -42,6 +42,46 @@ class StartsWithAcceptanceTest extends ExecutionEngineFunSuite with QueryStatist
     fNode = createLabeledNode("LABEL")
   }
 
+  test("should return null when END WITH is used on non-strings"){
```

Comment:
Nice tests, but maybe not a perfect fit for this test class. Rename this class maybe?